src/network/send.ts

```diff
@@ -344,7 +350,14 @@ export function newmsg(
       ...(includePhotoUrl && { photo_url: photoUrlToInclude }),
       // ...sender.contactKey && {contact_key: sender.contactKey}
     },
+  };
+  const personId = people.getPersonId();
+  if(personId){
+    result.sender.person = config.people_host + '/' + personId;
+    console.log("[+] person host full url ",  result.sender.person)
```

Comment:
we can use sphinxLogger.info here
```suggestion
    sphinxLogger.infoi(1[+] person host full URL  ${result.sender.person}`, logging.Network)
```